server/sonar-web/src/main/js/apps/issues/models/issues.js

```diff
@@ -96,14 +96,14 @@ export default Backbone.Collection.extend({
   },
 
   setIndex () {
-    return this.forEach(function (issue, index) {
+    return this.forEach((issue, index) => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)